src/index.html

```diff
@@ -3,12 +3,56 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header_inner">
+        <div class="img">
+          <a href="#home">
+            <img
+              src="./images/logo.png"
+              class="logo"
+              alt="logo"
+            >
+          </a>
+        </div>
+        <nav class="nav">
+          <ul class="nav_list">
+            <li>
+              <a class="is-active" href="#">Apple</a>
+            </li>
+            <li>
+              <a class="nav_link" href="#">Samsung</a>
+            </li>
+            <li>
+              <a class="nav_link" href="#">Smartphones</a>
+            </li>
+            <li>
+              <a class="nav_link"
+                data-qa="hover" href="#"
+              >
+                Laptops & Computers
+              </a>
+            </li>
```

Comment:
```suggestion
          <ul class="nav_list">
            <li>
              <a class="is-active" href="#">Apple</a>
            </li>
            
            <li>
              <a class="nav_link" href="#">Samsung</a>
            </li>
            
            <li>
              <a class="nav_link" href="#">Smartphones</a>
            </li>
            
            <li>
              <a class="nav_link"
                data-qa="hover" href="#"
              >
                Laptops & Computers
              </a>
            </li>
            
```
empty line between siblings will improve readibility